example/room-bot.ts

```diff
@@ -275,8 +270,7 @@ function manageDingRoom() {
       log.info('Bot', 'Room EVENT: topic - changed from %s to %s by member %s'
           , oldTopic
           , topic
-          , changer.name()
-        )
+          , changer.name())
```

Comment:
keep the original position of `)`, just add a comma.

Will not comment on this question anymore.